mscore/musescore.cpp

```diff
@@ -1087,6 +1089,7 @@ MuseScore::MuseScore()
       menuFormat->addSeparator();
 
       menuFormat->addAction(getAction("reset-beammode"));
+      menuFormat->addAction(getAction("reset-groupings"));
```

Comment:
This might be better placed in the Tools menu after Respell Pitches. Maybe a more concise name, too, like "Regroup Durations" or something.